app-main/public/js/lessons/index.js

```diff
@@ -221,6 +221,13 @@ export function addLesson() {
     lessonInput.value = "";
     renderLessons(user);
   }
+      let submitLessonElement=document.getElementById("submit");
```

Comment:
Would it make sense to place the logic within the `handleClear` function [here](https://github.com/jrobind/dev-jot/blob/master/app-main/public/js/events/eventHandlers.js#L13)? 

Also, I've tested this and it still appears to be broken when editing a lesson and clicking clear?

Reply:
It's working,  I have checked it. Let me send a gif.

Reply:
![video-to-gif-converter](https://user-images.githubusercontent.com/60141545/109961962-a5d3c500-7d10-11eb-860d-eb905726a035.gif)

Reply:
Awesome! This looks good. Thanks for sharing the gi